Dockerfile

```diff
@@ -4,6 +4,7 @@ RUN apt update && apt upgrade
 RUN apt install python3 -y
 RUN apt install python3-pip -y
 RUN apt install wget -y
+RUN apt install vim -y
```

Comment:
I'm being a bit picky, but I'm not sure we should include anything in the image that is just there for development convenience, since everything in the image will be scanned for compliance

Reply:
ok no worries, I should have asked first. I'll take it off